server/valgrind-out.txt

```diff
@@ -1,131 +1,148 @@
+--9679-- REDIR: 0x49156a0 (libc.so.6:strlen) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x49158d0 (libc.so.6:strncpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x4916950 (libc.so.6:stpcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x49163f0 (libc.so.6:memchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x4915b00 (libc.so.6:strspn) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x49167d0 (libc.so.6:mempcpy) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x4916ac0 (libc.so.6:strncasecmp) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x4917f80 (libc.so.6:rawmemchr) redirected to 0x483f220 (_vgnU_ifunc_wrapper)
+--9679-- REDIR: 0x4a0a610 (libc.so.6:__strrchr_avx2) redirected to 0x484e810 (rindex)
+--9679-- REDIR: 0x4a0df80 (libc.so.6:__memset_avx2_unaligned_erms) redirected to 0x4852770 (memset)
+--9679-- REDIR: 0x49120a0 (libc.so.6:malloc) redirected to 0x4848820 (malloc)
+--9679-- REDIR: 0x49123e0 (libc.so.6:free) redirected to 0x484b210 (free)
+--9679-- REDIR: 0x4913520 (libc.so.6:calloc) redirected to 0x484d9d0 (calloc)
+--9679-- REDIR: 0x4a0a400 (libc.so.6:__strchrnul_avx2) redirected to 0x4853330 (strchrnul)
+--9679-- REDIR: 0x4a0a7e0 (libc.so.6:__strlen_avx2) redirected to 0x484ed10 (strlen)
+--9679-- REDIR: 0x4a0d7c0 (libc.so.6:__memcpy_avx_unaligned_erms) redirected to 0x4852880 (memmove)
+--9679-- REDIR: 0x4a0bcb0 (libc.so.6:__strcpy_avx2) redirected to 0x484ee60 (strcpy)
+--9679-- REDIR: 0x4915fc0 (libc.so.6:__GI_strstr) redirected to 0x4853ae0 (__strstr_sse2)
+--9679-- REDIR: 0x4a06a80 (libc.so.6:__memcmp_avx2_movbe) redirected to 0x4852010 (bcmp)
+--9679-- REDIR: 0x4a05940 (libc.so.6:__strcmp_avx2) redirected to 0x484fbd0 (strcmp)
+--9679-- REDIR: 0x4a0d780 (libc.so.6:__mempcpy_avx_unaligned_erms) redirected to 0x4853440 (mempcpy)
+--9679-- REDIR: 0x4912740 (libc.so.6:realloc) redirected to 0x484dc50 (realloc)
+--9679-- REDIR: 0x4a062c0 (libc.so.6:__memchr_avx2) redirected to 0x484fd50 (memchr)
+--9679-- REDIR: 0x4a0ac30 (libc.so.6:__strcat_avx2) redirected to 0x484ea20 (strcat)
+==9679== 
+==9679== HEAP SUMMARY:
+==9679==     in use at exit: 3,264 bytes in 12 blocks
+==9679==   total heap usage: 270 allocs, 258 frees, 590,658 bytes allocated
+==9679== 
+==9679== Searching for pointers to 12 not-freed blocks
+==9679== Checked 100,735,120 bytes
+==9679== 
+==9679== 3,264 bytes in 12 blocks are possibly lost in loss record 1 of 1
+==9679==    at 0x484DA83: calloc (in /usr/libexec/valgrind/vgpreload_memcheck-amd64-linux.so)
+==9679==    by 0x40147D9: calloc (rtld-malloc.h:44)
+==9679==    by 0x40147D9: allocate_dtv (dl-tls.c:375)
+==9679==    by 0x40147D9: _dl_allocate_tls (dl-tls.c:634)
+==9679==    by 0x49027B4: allocate_stack (allocatestack.c:430)
+==9679==    by 0x49027B4: pthread_create@@GLIBC_2.34 (pthread_create.c:647)
+==9679==    by 0x10A924: main (aesdsocket.c:430)
```

Comment:
I think this is pointing to a real issue regarding conThread, see notes in review.